backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -128,7 +128,7 @@ public Parcel Get(int id)
                     (isAdmin || !p.IsSensitive || (viewSensitive && userAgencies.Contains(p.AgencyId)))) ?? throw new KeyNotFoundException();
 
             // Remove any sensitive buildings from the results if the user is not allowed to view them.
-            parcel?.Buildings.RemoveAll(b => b.IsSensitive && !userAgencies.Contains(b.AgencyId));
+            parcel?.Buildings.RemoveAll(b => !isAdmin && b.IsSensitive && !userAgencies.Contains(b.AgencyId));
```

Comment:
Corrected issue where an administrator would not be able to see sensitive buildings.